docs/src/operate/lifecycle/deregister.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Deregister entities
+tags: [Lifecycle]
+sidebar_position: 1
+description: Deregister entities from thin-edge
+---
+
+# Deregister entities
+
+All the entities registered with thin-edge are stored with the MQTT broker as retained messages,
+with their metadata spread across multiple topics.
+For e.g, for a child device `child01`, the registration message is stored on the `te/device/child01//` topic,
+its twin data is stored across several `te/device/child01///twin/<twin-data-type>` topics,
+its command metadata stored across several `te/device/child01///cmd/<cmd-type>` topics and so on.
+
+On top of that, when a device has services associated with it, or has nested child devices,
+they all have their own respective registration topics and other metadata topics.
+So, de-registering a device involves de-registering itself, its metadata and
```

Comment:
```suggestion
So, deregistering a device involves deregistering itself, its metadata and
```

Reply:
Keeping `de-registering` to satisfy the typo checker even though `deregistering` also seems generally valid.